ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -305,6 +305,37 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-sdn-mtu-migrate
+- as: aws-upi-baselinecaps-none-amd-f14
```

Comment:
I guess we can do similar things for 4.13, because aws multi got GA  release on 4.13.

Reply:
done, move job `aws-ipi-disconnected-private-sno-baselinecaps-none` to multi nightly job config, change `aws-ipi-baselinecaps-none` to `aws-upi-baselinecaps-none` and move to multi nightly job config.

Reply:
> move job aws-ipi-disconnected-private-sno-baselinecaps-none to multi nightly job config

Compared with 4.14+, seem like no such job to cover, are we doing that on purpose?

Reply:
keep job `aws-ipi-disconnected-private-sno-baselinecaps-none` on 4.12 and 4.13, because
1.  it contains step `ipi-conf-manual-creds-remove-unnecessary-creds` to make sure that cluster with baselinecaps:none + cco manual works well
2.  cover sno + baselinecaps: none

On 4.14+, the step `ipi-conf-manual-creds-remove-unnecessary-creds` is covered by `aws-ipi-disconnected-sts-basecap-none`, and we added Jenkins CI profile 136_sno-packet-etcd_encryption-realtime_kernel-baselinecaps_none to cover sno case, so removed `aws-ipi-disconnected-private-sno-baselinecaps-none`

Reply:
Thanks for the thoughtful coverage.